keps/sig-multicluster/4322-cluster-inventory/README.md

```diff
@@ -524,12 +524,162 @@ minimum kubelet version, maximum kubelet version, and enabled featureset version
 
 #### Properties
 
-Name/value pairs to represent properties of the clusters. It could be a
-collection of ClusterProperty resources, but could also be info based on
+Name/value pairs to represent properties of the clusters. One possible usage of properties is 
+to help another controller to make workload orchestration decisions. They could be 
+a set of [About-API](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/2149-clusterid)
+ClusterProperty resources collected from the member clusters, but could also be info based on
 other implementations. The name of the cluster property can be predefined
 name from ClusterProperty resources and is allowed to be customized by
 different cluster managers.
 
+For all the about-api ClusterProperty that clusterProfile would honor, it MUST have a label with key “multicluster.x-k8s.io/clusterProfile” and value of “true”.
+
+Here are a few ClusterProperty that the clusterProfile API would also support. 
+
+##### cluster-endpoints.k8s.io
+It is a list of the url that can reach the api-server of the k8s cluster
+  * All clusterProfile API implementation MUST have this property.
+  * It could contain IP addresses too
+  
+Here is an example of a cluster-endpoints.k8s.io ClusterProperty
+```yaml
+apiVersion: about.k8s.io/v1
+kind: ClusterProperty
+metadata:
+  name: cluster-endpoints.k8s.io
+  labels:
+    multicluster.x-k8s.io/clusterProfile: “true”
+spec:
+  value: ["100.3.3.4:5683","qs-oar7gr9p.azmk8s.io:443"]
+```
+Here is the corresponding ClusterProfile API
+```yaml
+properties:
+   - name: cluster-endpoints.k8s.io
+     value:  ["100.3.3.4:5683","qs-oar7gr9p.azmk8s.io:443"]
+```
+
+##### location.topology.k8s.io
+It contains the location of the k8s cluster
```

Comment:
I think that it makes more sense to recognize that cluster locations exist in a hierarchy, similar to the way that node locations exist in a hierarchy.

Reply:
there are also the officially defined `topology.k8s.io` label keys which may be relevant when thinking about naming/extensibility: https://github.com/kubernetes/enhancements/tree/master/keps/sig-architecture/1659-standard-topology-labels